ios/ReactNativeBoilerplate/Info.plist

```diff
@@ -22,6 +22,8 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>CodePushDeploymentKey</key>
+	<string>$(CODEPUSH_KEY)</string>
```

Comment:
I don't think this is a correct description: https://github.com/Vivify-Ideas/react-native-boilerplate/pull/17/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R92

We should state exactly where the variable will be added because we have different envs, so staging and prod code push keys are different...

Reply:
Same goes for android